Gems/Atom/Feature/Common/Assets/Materials/Types/EnhancedPBR.materialtype

```diff
@@ -1223,14 +1223,53 @@
                     "min": 0.0,
                     "softMax": 20.0
                 },
+                {
+                    "name": "shrinkFactor",
+                    "displayName": "    Shrink Factor",
+                    "description": "Shrink (absolute) offset towards the normal opposite direction to ensure correct shadow map projection",
+                    "type": "float",
+                    "defaultValue": 0.005,
+                    "min": 0.0,
+                    "softMax": 0.05,
+                    "connection": {
+                        "type": "ShaderInput",
+                        "name": "m_shrinkFactor"
+                    }
+                },
+                {
+                    "name": "angleOffset",
+                    "displayName": "    Angle Offset",
+                    "description": "cosine of angle to extend below (N . L = 0) in scattering through thin objects",
+                    "type": "float",
+                    "defaultValue": 0.1,
+                    "min": -1.0,
+                    "softMax": 1.0,
+                    "connection": {
+                        "type": "ShaderInput",
+                        "name": "m_angleOffset"
+                    }
+                },
+                {
+                    "name": "distanceAttenuation",
```

Comment:
I think it would not be the same, since a sphere light shadow map (e.g.) will always have the same resolution, independently of the object's projected area into the shadow map (which is what we would like to take into account IMO, and what the distance attenuation factor approximately achieves).

Still, an additional attenuation factor could be derived from the shadow map resolution, I agree. It could avoid showing 'ugly' results at low-res configurations.